### PR TITLE
Add ivanceras/sauron-native as GUI and ivanceras/titik as TUI

### PR DESCRIPTION
**Pull request by ivanceras**

### CODE REVIEW

**Comment by palfrey** [comment]
I'm a little concerned about the maturity of these packages. As I wasn't seeing any CI, I checked out https://github.com/ivanceras/titik and tried doing a simple `cargo test` and got
```
Updating crates.io index
error: failed to get `ito-canvas` as a dependency of package `titik v0.1.0 (/Users/palfrey/src/titik)`

Caused by:
  failed to load source for dependency `ito-canvas`

Caused by:
  Unable to update /Users/palfrey/src/ito-canvas

Caused by:
  failed to read `/Users/palfrey/src/ito-canvas/Cargo.toml`

Caused by:
  No such file or directory (os error 2)
```
I'm not entirely convinced at the moment that they're ready for inclusion, but if you're planning on resolving those sorts of problems soon, they might be of interest.

**Comment by ivanceras** [comment]
The master branch is under heavy development in the mean time.
The published [crate of titik](https://crates.io/crates/titik) however is in a workable state.

Currently modularizing functionalities into smaller crate. One of the dependencies is not yet linked properly.
A quick workaround is checking it out manually at https://github.com/ivanceras/ito-canvas

I'll setup a CI soon as I can

**Comment by ivanceras** [comment]
@palfrey It's fixed now. Please merge

**Comment by palfrey** [comment]
Thanks for adding the CI. Now you have that, can you please add the build badges for each of the repositories. Thanks!

File: README.md [hunk 1]
```diff
@@ -1213,7 +1214,7 @@ See also [Are we game yet?](https://arewegameyet.rs)
 * [Sciter](https://sciter.com/)
   * [sciter-sdk/rust-sciter](https://github.com/sciter-sdk/rust-sciter) — Sciter bindings [<img src="https://ci.appveyor.com/api/projects/status/github/sciter-sdk/rust-sciter?svg=true">](https://ci.appveyor.com/project/sciter-sdk/rust-sciter)
 * [hecrj/iced](https://github.com/hecrj/iced) — A cross-platform GUI library for Rust focused on simplicity and type-safety. Inspired by Elm.
-
+* [ivanceras/sauron-native](https://github.com/ivanceras/sauron-native) - A truly native and cross platform GUI library. One unified code can be run as native GUI, Html Web and TUI.
```
Comment (palfrey) [inline README.md:1217]: This one is still missing the travis badge

File: README.md [hunk 2]
```diff
@@ -1436,6 +1437,7 @@ See also [Are we web yet?](http://www.arewewebyet.org) and [Rust web framework c
   * [seed](https://seed-rs.org/) — A Rust framework for creating web apps
   * [stdweb](https://crates.io/crates/stdweb) — A standard library for the client-side Web [![Build Status](https://api.travis-ci.org/koute/stdweb.svg?branch=master)](https://travis-ci.org/koute/stdweb)
   * [yew](https://crates.io/crates/yew) — Rust framework for making client web apps
+  * [sauron](https://github.com/ivanceras/sauron) - Client side web framework which closely adheres to The Elm Architecture. [![Build Status](https://travis-ci.org/ivanceras/sauron.svg?branch=master)](https://travis-ci.org/ivanceras/sauron)
```
Comment (palfrey) [inline README.md:1440]: The image link should be https://api.travis-ci.org/ivanceras/sauron.svg?branch=master as the current one 302's to that

Comment (ivanceras) [reply README.md:1440]: I can't reproduce the 302 error. Please check again.

File: README.md (hunk 1 above)
Comment (ivanceras) [reply README.md]: I added a CI for sauron-native on the latest commit.

File: README.md (hunk 2 above)
Comment (palfrey) [reply README.md:1440]: So, it's actually a 301. `curl -v https://travis-ci.org/ivanceras/sauron.svg?branch=master` gets a `HTTP/1.1 301 Moved Permanently` with ` Location: https://api.travis-ci.org/ivanceras/sauron.svg?branch=master`

Comment (ivanceras) [reply README.md:1440]: Ah thanks, I didn't realize that by just opening the links using the browser.

**Comment by palfrey** [comment]
Thanks for all your work here. Now merging.

**Pull request merged**